astropy/time/core.py

```diff
@@ -594,7 +597,11 @@ def light_travel_time(self, skycoord, kind='barycentric', location=None):
             location = self.location
 
         from ..coordinates import (UnitSphericalRepresentation, CartesianRepresentation,
-                                   HCRS, ICRS, GCRS, EarthLocation, SkyCoord)
+                                   HCRS, ICRS, GCRS, solar_system_ephemeris)
+
+        # get the ephemeris for later (we don't want to set it to None)
+        if ephemeris is None:
```

Comment:
This slightly begs the question whether we're not doing the wrong thing for `solar_system_ephemeris.set()`. Looking at the `ScienceState` code, we could `None` stand for "don't change anything" by using
```
    @classmethod
    def validate(cls, value):
        if value is None:
            return cls._value
        # Set up Kernel; if the file is not in cache, this will download it.
        cls.get_kernel(value)
        return value
```